ksql-common/src/main/java/io/confluent/ksql/util/KsqlConstants.java

```diff
@@ -48,4 +48,8 @@
 
   public static final String DOT = ".";
   public static final String STRUCT_FIELD_REF = "->";
+
+  public static final String AVRO_SCHEMA_NAMESPACE = "io.confluent.ksql.avro_schemas";
```

Comment:
is this the same as what we previously used? I'm not finding `avro_schemas` in the older code, but if this is a regression, we should be restoring the existing behavior.

Reply:
4.1 used the namespace `ksql` and the name `avro_schema`. 5.0 isn't meant to be compatible with 4.1, so it should be ok to change the name here. The regression being fixed is that previously a user could use the class generated by avro from the schema, and currently in 5.0 a user cannot unless their whole app is in the default package.